IntelliTect.TestTools.TestFramework/IntelliTect.TestTools.TestFramework.Tests/TestBase.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Xunit;
+
+namespace IntelliTect.TestTools.TestFramework.Tests
+{
+    public class TestBase
+    {
+        protected static void ValidateAggregateException(AggregateException result, int expectedInnerExceptions, params string[] messages)
+        {
+            if(result is null) throw new ArgumentNullException(nameof(result));
```

Comment:
Now that your project is net6 you can use this goodness for null checks that don't include an assignment.

```suggestion
            ArgumentNullException.ThrowIfNull(result);
```

Reply:
@Keboo I still get errors around the object being possibly null even after changing to the ThrowIfNull method. E.G. in this case, on the very next line the result.InnerExceptions.Count invocation gets flagged as requiring validation that 'result' is not null. Am I missing something?